test/e2e/app-dir/next-font/next-font.test.ts

```diff
@@ -52,8 +45,12 @@ describe('app dir - next/font', () => {
 
               // layout
               expect(JSON.parse($('#root-layout').text())).toEqual({
-                className: expect.stringMatching(/^__className_.{6}$/),
```

Comment:
do we not want to keep the more specific regex for webpack?

Reply:
Tried to match them a bit more, overall the format really doesn't matter as long as the class is there